README.md

```diff
@@ -0,0 +1,60 @@
+# perfgrind
+
+This is 'perfgrind', tools for collecting samples from Linux performance events
+subsystem and converting profiling data to 'callgrind' format.
```

Comment:
```suggestion
subsystem and converting profiling data to `callgrind` format.
```

Reply:
Again: neither code nor something to enter. We could move from ' to " here if you like